src/ChannelServer/Network/ChannelPacketHandler.cs

```diff
@@ -836,5 +870,71 @@ public void CZ_CHECK_PING(ChannelConnection conn, Packet packet)
 		{
 			// No parameters, no response.
 		}
+
+		/// <summary>
+		/// This packet is used to cast skills in the ground
+		/// </summary>
+		/// <param name="conn"></param>
+		/// <param name="packet"></param>
+		/// <example>
+		/// 
+		/// </example>
+		[PacketHandler(Op.CZ_SKILL_GROUND)]
+		public void CZ_SKILL_GROUND(ChannelConnection conn, Packet packet)
+		{
+			var unk1 = packet.GetByte();
+			var skillId = packet.GetInt();
+			var unk2 = packet.GetInt();
+			var x1 = packet.GetFloat();
+			var y1 = packet.GetFloat();
+			var z1 = packet.GetFloat();
+			var x2 = packet.GetFloat();
+			var y2 = packet.GetFloat();
+			var z2 = packet.GetFloat();
+			var cos = packet.GetFloat();
+			var sin = packet.GetFloat();
+			var unk3 = packet.GetInt();
+			var unk6 = packet.GetByte();
+			var unk7 = packet.GetByte();
+
+			Log.Warning("CZ_SKILL_GROUND");
+			Log.Warning("unk1: {0}", unk1);
+			Log.Warning("skillId: {0}", skillId);
+			Log.Warning("unk2: {0}", unk2);
+			Log.Warning("Pos1: {0},{1},{2}", x1, x1, z1);
+			Log.Warning("Pos2: {0},{1},{2}", x2, y2, z2);
+			Log.Warning("Dir: {0},{1}", cos, sin);
+			Log.Warning("unk3: {0}", unk3);
+			Log.Warning("unk6: {0}", unk6);
+			Log.Warning("unk7: {0}", unk7);
+
+			var character = conn.SelectedCharacter;
+
+			// Check parameters
+			var packetPosition1 = new Shared.World.Position(x1, y1, z1);
```

Comment:
You don't need to include the full namespace every time, add a using for Shared.World instead.